packages/site/pages/components/textinput.js

```diff
@@ -72,13 +72,6 @@ export default _ => (
               horizontal icon placement (from <code>TextInput.iconAligns</code>)
             </span>
           ]),
```

Comment:
Should the dual ref ability be documented?

Reply:
yeah i should add that